python/ray/serialization.py

```diff
@@ -291,7 +296,12 @@ def _deserialize_object(self, data, metadata, object_ref):
                     object_ref.hex(), object_ref.owner_address(), object_ref.call_site()
                 )
             elif error_type == ErrorType.Value("RUNTIME_ENV_SETUP_FAILED"):
-                return RuntimeEnvSetupError()
+                error_info = self._deserialize_error_info(data, metadata_fields)
+                # TODO(sang): Assert instead once actor also reports error messages.
```

Comment:
If we assert now, it fails when actors are failed with runtime env (because it is not reported yet)

Reply:
Sorry I didn't understand, what would we assert?

Reply:
ah sorry assert that the erorr_info always has the correct field.